docs/getting_started/downloading_files.md

```diff
@@ -0,0 +1,23 @@
+#Downloading Files
+
+When a task has been completed, the output data from that job is made available to download. Files can be downloaded by using the Desktop Companion or the Conductor command-line interface (CLI).

```

Comment:
We should probably be more consistent with how we talk about the app. On the marcom site, and our outgoing communications, I've been referring to it as Conductor Companion, Companion, or using lowercase when describing as a companion app/ desktop companion app

Reply:
I'll update all the references to say Conductor Companion